t/02-pod.t

```diff
@@ -4,4 +4,6 @@ use Test::Most;
 eval 'use Test::Pod';
 plan skip_all => "Test::Pod required for testing POD" if $@;
 
+fail 'induce fail and check if we can see that in circleci';
```

Comment:
This is a bad idea I think, [all_pod_files_ok](https://metacpan.org/pod/Test::Pod#all_pod_files_ok(-%5B@entries%5D-)) defines the plan, so the final count doesn't match

Reply:
does not matter. I just wanted to induce a fail. In any case I can see that https://circleci.com/gh/os-autoinst/openQA/41915?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link shows the failed test output as I wanted to have.

Reply:
Ack.

For the record `/usr/lib/perl5/5.30.1/TAP/Parser/Iterator/Process.pm` is where `merge` is actually implemented after the option is passed down from `prove` and `SourceHandler` respectively. The code relevant for us is this:

```perl
            $err = $merge ? '' : IO::Handle->new;
            eval { $pid = open3( '<&STDIN', $out, $err, @command ); };
            die "Could not execute (@command): $@" if $@;
            $sel = $merge ? undef : IO::Select->new( $out, $err )
```

Reply:
Cool, you found it! I guess in theory we could try to extend that to take everything that is captured from `$err` and write that elsewhere so that we can again see the output in the normal log view despite all output also being written into the dumped archive file. But I guess for now we benefit from removing the merge parameter again to have at least the error output in the normal log view.